src/main.rs

```diff
@@ -234,14 +263,18 @@ async fn main() {
     let app_state = AppState { database: db_data };
 
     let app = Router::new()
-        .route("/:uuid", get(return_config_file))
-        .route("/", post(redirect))
+        .route("/:uuid", get(return_config_file_get))
+        .route("/", get(browser_html))
+        .route("/", post(redirect_post_mulipart_form))
```

Comment:
You are now kind of deleting the old way of having the possibility to just do a a simple curl like `curl -L http://$server -d @wicked.xml` which I think is still a good tool to have IMO. I would maybe let the `/` post route allow this simple xml way so it can be easily called via the curl command and introduce an alternative route for this multipart function (which will probably most of the time only be called via the web interface anyway so a more "difficult" path should be fine).